webapp/src/main/java/org/opentestsystem/rdw/admin/repository/impl/JdbcGroupRepository.java

```diff
@@ -57,8 +61,24 @@
         );
     }
 
+    // TODO: This implementation only updates the isDeleted field.
+    // TODO: Other fields need to be validated before they can be updated.
+    @Override
+    public void update(@NotNull final PermissionScope permissionScope, @NotNull final Group group) {
```

Comment:
@malaffoon and I discussed this.  Since the UI will eventually have to become aware of the soft delete by allowing an "undelete", then a PUT would seem to work best since you can use that same PUT to do the undelete, instead of having to implement a `groups/id/undelete` as a separate api route.